### PR TITLE
Fixes VAO picking

### PR DESCRIPTION
**Pull request by stephomi**

### CODE REVIEW

**Comment by stephomi** [comment]
The changes of apply( this ) to call( this ) (so without arguments) is not about performance but about semantics.
call is more specific than apply so it should be the prefered version.

File: sources/osg/Geometry.js [hunk 1]
```diff
@@ -328,7 +328,10 @@ Geometry.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit( No
                 ];
                 for ( j = 0, m = listVABuff.length; j < m; j++ ) {
                     vaoSetup.push( '  va = vaList[ ' + j + '];' );
-                    vaoSetup.push( '  if ( va.isDirty() ) va.compile( gl );' );
+                    vaoSetup.push( '  if ( va.isDirty() ) {;' );
+                    vaoSetup.push( '    va.bind( gl );' );
```
Comment (stephomi) [inline sources/osg/Geometry.js]: the buffer should be bound before re-compiling it.

**Comment by stephomi** [comment]
maybe same issue here https://github.com/cedricpinson/osgjs/blob/develop/sources/osg/Geometry.js#L225

**Comment by cedricpinson** [comment]
Yeah same issue, could you fix it too plz ?

**Comment by stephomi** [comment]
k done

**Pull request merged**